src/components/ConfirmationModal/ConfirmationModal.js

```diff
@@ -0,0 +1,55 @@
+import { useEscape } from '../../hooks/useEscape';
+import './ConfirmationModal.css';
+
+function ConfirmationModal({
+  isActive,
+  closeModal,
+  buttonText,
+  title,
+  handleButton,
+}) {
+  useEscape(handleCloseModal);
+
+  const handleClickOutsideClose = (evt) => {
+    if (evt.target.classList.contains('modal')) {
+      handleCloseModal();
+    }
+  };
+
+  if (isActive) {
+    setTimeout(() => {
+      document.querySelector('.modal').classList.add('modal_opened');
+    }, 10);
+  }
+
+  function handleCloseModal() {
```

Comment:
You should use the same logics of closing like you do with any other modal in the project. You have `closeModal` for closing any modal. 


```js

  function handleCloseModal() {
    closeModal();
  }
  

```


So, you don't need `handleCloseModal` because you have `closeModal`